api/config/cdr_config_local.json

```diff
@@ -22,6 +22,20 @@
     }
   ],
   "cdrVersions": [
+    {
+      "cdrVersionId": -1,
+      "name": "Dummy version for Spring initialization - do not use",
```

Comment:
I don't mind the idea of having a dummy version server side, if that's the cleanest solution - but I dislike putting this into the config; feels like an unnecessary leaking of abstractions.

If our Spring initialization can't run without a dummy value, it feels like the onus should be on the Spring initialization code to solve the problem. Can you comment a bit on where the sticking points are here? Lets say we instead:

1. Just threw an error instead, when the default CDR is requested
2. or, just returned null if no CDR version context is configured

What breaks?

Alternatively:

3. Creating a stub CdrVersion object on the server (that never gets stored / saved)

Reply:
1 is essentially https://github.com/all-of-us/workbench/pull/6231 - I see NPEs and stack traces on dev-up.